projects/kubernetes/e2e/Makefile

```diff
@@ -0,0 +1,29 @@
+ORG?=linuxkitprojects
+IMAGE=kubernetes-e2e
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
+
+kubernetes_version=v1.8.0
+
+kube_release_artefacts=https://dl.k8s.io/$(kubernetes_version)/bin/linux/amd64
+
+.PHONY: build
+build: _cache/kubernetes-test.tar.gz _cache/kubectl
+	docker build --network=none -t $(ORG)/$(IMAGE):$(HASH) .
```

Comment:
Please can we switch the actual build phase over to the `linuxkit pkg build` scheme. I appreciate this will still need wrapping in a `Makefile` to fetch deps etc.

I thought I'd mentioned this before but clearly I forgot, sorry.